backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/UserPermission.java

```diff
@@ -2,16 +2,20 @@
 
 /**
  * Permissions that a user may hold, usually via their {@link OrganizationRole}.
+ *
+ * DO: Keep updated with UserPermission in schema.graphqls
  */
 public enum UserPermission {
     READ_PATIENT_LIST,
+    READ_PATIENT_INCL_DEL_LIST,
     SEARCH_PATIENTS,
-    READ_RESULT_LIST,
     EDIT_PATIENT,
+    DELETE_PATIENT,
+    READ_RESULT_LIST,
     EDIT_FACILITY,
     EDIT_ORGANIZATION,
     START_TEST,
     UPDATE_TEST,
     SUBMIT_TEST,
-    EXPORT_TEST_EVENT;
```

Comment:
My IDE insists putting a `;` here is wrong.

Reply:
Fix your IDE?

Reply:
The semicolon is only needed when fields / methods follow. For a simple list of constants, it's just non-standard.

But I can turn down the `Warn for unnecessary semicolon` to a warning, not an error.

> Reports on any unnecessary semicolons, whether between class members, inside block statements, or after class definitions. While valid Java, these semicolons are redundant, and may be removed.